tensorboard/data/experimental/test_binary.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A test binary that can be used to test ExperimentFromDev features."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+
+from tensorboard.data.experimental import experiment_from_dev
+
+
+def main(args):
+    experiment = experiment_from_dev.ExperimentFromDev(
+        args.experiment_id, api_endpoint=args.api_endpoint
+    )
+    dataframe = experiment.get_scalars()
+    print(dataframe)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser("Test run of ExperimentFromDev")
```

Comment:
Let’s try to keep all the logic—and in particular, all assignments—in a
function (fine to put this in `main`) so we don’t have to worry about
Python scoping issues.